meta/runtime.yml

```diff
@@ -283,6 +287,8 @@ plugin_routing:
       deprecation:
         removal_version: 3.0.0
         warning_text: see plugin documentation for details
+    ome_device_info:
+      redirect: dellemc.openmanage.ome_device_info
     one_image_facts:
       deprecation:
```

Comment:
You also need to add redirects for the module_utils.

Reply:
completed